_inc/connect-button.js

```diff
@@ -1,6 +1,55 @@
+/* global jpConnect */
+
 jQuery( document ).ready( function( $ ) {
 	$( '.jp-connect-button' ).click( function( event ) {
 		event.preventDefault();
-		alert( '❤️❤️❤️❤️❤️❤️❤️❤️❤️❤️' );
+		if ( ! jetpackConnectButton.isRegistering ) {
+			jetpackConnectButton.handleClick();
+		}
 	} );
+	var jetpackConnectIframe = $( '<iframe class="jp-jetpack-connect__iframe" />' );
+
+	var jetpackConnectButton = {
+		isRegistering: false,
+		handleClick: function() {
+			jetpackConnectButton.isRegistering = true;
+			$( '.jp-connect-button' ).text( jpConnect.buttonTextRegistering );
```

Comment:
Should we disable the button after clicking and changing the label to 'Loading'? Something like this:

```js
$( '.jp-connect-button' ).attr( 'disabled', true );
```

This way the connection flow here would match analogical flows elsewhere, e.g. in Calypso.